pkg/console/network.go

```diff
@@ -16,6 +17,24 @@ import (
 	"github.com/harvester/harvester-installer/pkg/config"
 )
 
+func isDefaultRouteExist() bool {
+	routes, err := netlink.RouteList(nil, syscall.AF_INET)
+	if err != nil {
```

Comment:
This is also a possible return path, the error message can only be checked on log instead of installation panel.

Please consider to return `bool, error` and capture the error on the caller to show it. thanks.